pages/studentPortal/labs/multiplication-connect/Index.tsx

```diff
@@ -21,11 +21,54 @@ const createGrid = () => {
   return grid;
 }; 
 
+export const calculateWinner = (grid) => {
```

Comment:
This looks good to me, but I think you should move this function into the api folder and write a test for this function

We can chat about how to do that tmr